vue-client/src/components/inspector/language-entry.vue

```diff
@@ -119,6 +105,21 @@ export default {
       });
     },
   },
+  mounted() {
+    if (this.tag !== 'none') {
+      this.$emit('addToCache');
+
+      if (this.supportedTags.indexOf(this.tag) === -1) {
```

Comment:
Use `!this.supportedTags.includes(this.tag)` instead? Reads better in IMO!

Reply:
Updated!